src/binaries/BinariesManager.ts

```diff
@@ -19,24 +18,16 @@ export class BinariesManager {
                                      accountContextManager: AccountContextManager,
                                      expectedCLIVersion: string
     ): IBinariesUtility {
-        if (this.useBinariesUtilityV2()) {
-            logger.trace(TelemetryEvent.BinariesUtility_VersionV2);
+        if (this.useBinariesUtility()) {
+            logger.trace(TelemetryEvent.BinariesUtility_Version);
             return new BinariesUtilityV2(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
         }
-
-        logger.trace(TelemetryEvent.BinariesUtility_VersionV1);
-        return new BinariesUtility(logger, context, commandEnvironmentVariables, accountContextManager, expectedCLIVersion);
     }
 
-    private static useBinariesUtilityV2(): boolean {
+    private static useBinariesUtility(): boolean {
```

Comment:
💡 nice, looks good, I can try add some UT (Unit tests for these pieces) tomorrow and we can try them out.

Suggestion I wonder if we can simplify this: 

* https://github.com/Azure/vscode-bridge-to-kubernetes/blob/e53fc61f1eeaafd4ddf7763a09ab37a70cd2ad1f/src/utility/VersionUtility.ts#L51 

especially the nested `if` please hav a play around. Thanks.

Reply:
yeah, I'm working on simplifying the versionUtility (the nested if conditions)